test/relations.test.js

```diff
@@ -112,7 +115,7 @@ describe('relations', function() {
           book.chapters.create(chapters, function(err, chs) {
             should.not.exist(err);
             should.exist(chs);
-            chs.should.have.lengthOf(chapters.length);
+            chs.should.have.lengthOf(book.chapters.length);
```

Comment:
Why is this change needed? If `book.chapters` has different length than`chapters`, then you are subtly changing the test purpose in a way that's not clear to me (and probably to other future readers too).